Gems/Pointcloud/Code/Source/Render/PointcloudFeatureProcessor.h

```diff
@@ -90,5 +95,6 @@ namespace Pointcloud
         AZ::RHI::Ptr<AZ::RHI::ShaderResourceGroupLayout> m_drawSrgLayout; //!< Shader resource group layout for the draw packet
         AZStd::unordered_map<PointcloudHandle, PointcloudData> m_pointcloudData; //!< Map of pointcloud data
         PointcloudHandle m_currentPointcloudDataIndex = 0; //!< Index to the next pointcloud data to be created
+        AZStd::unordered_map<AZ::Data::AssetId, AZStd::pair<PointcloudHandle, AZ::Data::Asset<PointcloudAsset>>> m_pointcloudAssets;
```

Comment:
Why AZ::Data::Asset<PointcloudAsset>> is not a field in Pointcloud data?

Reply:
yaa I guess makes sense